tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -473,9 +473,9 @@ object TastyFormat {
   final val PARAMalias = 41
   final val TRANSPARENT = 42
   final val INFIX = 43
-  final val EMPTYCLAUSE = 44
-  final val SPLITCLAUSE = 45
-  final val INVISIBLE = 46
+  final val INVISIBLE = 44
```

Comment:
normally this would require a `MajorVersion` bump, but as we have not yet had a stable `ExperimentalVersion` this is fine